raspios/Dockerfile.customize

```diff
@@ -84,47 +49,54 @@ ADD overlay-rootfs /
 
 # Auto console login
 RUN systemctl set-default multi-user.target \
- && mkdir -p /etc/systemd/system/getty.target.wants \
- && ln -fs /lib/systemd/system/getty@.service /etc/systemd/system/getty.target.wants/getty@tty1.service \
- && systemctl enable getty@tty1.service
-
-# Configure pi account
-RUN chown -R pi /home/pi \
- && chmod 755 /home/pi \
- && echo "root:cogip" | chpasswd \
- && echo "pi:cogip" | chpasswd \
- && rm -f /etc/systemd/system/multi-user.target.wants/userconfig.service
+ && systemctl enable getty@tty1.service \
+ && systemctl enable serial-getty@ttyAMA0.service
 
 # Configure ssh server
-RUN chown pi:pi -R /home/pi/.ssh \
- && chmod 700 /home/pi/.ssh \
- && chmod 600 /home/pi/.ssh/* \
+RUN echo "root:cogip" | chpasswd \
  && chmod 600 /etc/ssh/ssh_host_* \
  && chmod 640 /etc/ssh/ssh_host_*.pub \
  && systemctl enable ssh
 
 # Configure Cockpit
 RUN chgrp cockpit-ws /etc/cockpit/cockpit.conf
 
-# Cleanup
-RUN apt-get clean
+# Use systemctl-networkd to manage network interfaces
+RUN systemctl disable networking
+RUN systemctl disable NetworkManager
+RUN systemctl disable NetworkManager-wait-online.service
```

Comment:
you could even try to remove it: `apt-get purge NetworkManager`

Reply:
There are so many packages that could be removed that I chose to not remove any.